elements/rh-button/package.json

```diff
@@ -8,7 +12,7 @@
   },
   "main": "index.js",
   "scripts": {
-    "build": "../../node_modules/.bin/gulp",
+    "build": "../../node_modules/.bin/gulp && ../../node_modules/.bin/rollup -c && ../../node_modules/.bin/prettier --ignore-path ../../.prettierignore --write '**/*.{js,json}'",
```

Comment:
@kylebuch8 This was a bit of a quick fix that we should revisit.  Ideally rollup-ing and prettier-ing would be done in the gulpfile, but it would require a larger refactoring of the gulpfile than I was comfortable with in an already-large PR.